### DISCUSSION TITLE
TypeError: getCurves is not a function

### DISCUSSION THREAD

**Discussion by jbblanton** [panva/jose#93] [Q&A]
I have been unable to make use of the Jose module. I started with `npm install jose` and have   `const { JWT } = require('jose');`  at the top of my code. No other code yet added to make use of the module, and I always get this error:

TypeError: getCurves is not a function
./node_modules/jose/lib/registry/ec_curves.js
```
   4 | 
   5 | const curves = new Set()
   6 | 
>  7 | if (getCurves().includes('prime256v1')) {
   8 |   curves.add('P-256')
   9 | }
  10 | 
```

I haven't found any explanations online, but would greatly appreciate some guidance on how to make this work.

Thank you

**Comment by panva** [comment]
https://github.com/panva/jose/search?q=getCurves+is+not+a+function&type=Issues

Use the signalled supported Node.js versions or don’t transpile crypto, one of these two.

**Comment by panva** [comment]
A quick note, i've release a new major (v3.x) which is now a universal module with 0 dependencies and now works the same in the browser as well as node.
